pallets/common/src/traits/identity.rs

```diff
@@ -24,7 +24,8 @@ use sp_std::vec::Vec;
 #[derive(codec::Encode, codec::Decode, Clone, PartialEq, Eq, Debug)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 pub enum LinkedKeyInfo {
-    Unique(IdentityId),
+    Master(IdentityId),
+    Signer(IdentityId),
```

Comment:
Let's not push this change to testnet v1. A lot of our tooling using the unique enum to fetch the linkage. Instead, we can check if a key is a master/signer by first fetching it's did and then fetching the master key of that did and then comparing the keys. It's not optimal but it maintains backward compatibility and given how close we are to v1, I'd like to avoid API breaking changes.